### PR TITLE
Support parens

### LINKED ISSUE #98: tag with parens doesnt work
when i have
```
const myQuery = gql`
  query Wow {
    whatever {
      id
    }
  }
```
then autocomplete works, and syntax highlight works, so i could go under `id` and type `la` and id get a `label` suggestion.

but with
```
const myQuery = gql(`
  query Wow {
    whatever {
      id
    }
  }
`)
```
which unfortunately is required for typescript and codegen to work, now autocomplete and syntax highlight doesnt work!

can you fix is so that with parens, both autocomplete and syntax highlight works?

### PR DESCRIPTION
**Pull request by pvinis**
Turns out not too hard to fix. i added the parens in the regex.

Fixes https://github.com/apollographql/vscode-graphql/issues/98.

### CODE REVIEW

File: src/language-server/__tests__/document.test.ts [hunk 1]
```diff
@@ -93,6 +93,33 @@ describe("extractGraphQLDocuments", () => {
       expect(documents?.[0].syntaxErrors.length).toBe(0);
       expect(documents?.[0].ast?.definitions.length).toBe(2);
     });
+
+    it("works with parens", () => {
+      const textDocument = mockTextDocument(`
+      gql(\`
```
Comment (pvinis) [inline src/language-server/__tests__/document.test.ts:99]: heres the main diff

File: src/language-server/__tests__/document.test.ts [hunk 2]
```diff
@@ -93,6 +93,33 @@ describe("extractGraphQLDocuments", () => {
...
+        \${Hero.fragments.character}
+
+        {
+          reviews(episode: NEWHOPE) {
+            ...ReviewList_reviews
+          }
+        }
+
+      \${ReviewList.fragments.reviews}
+    \`)
```
Comment (pvinis) [inline src/language-server/__tests__/document.test.ts:115]: and here

**Comment by pvinis** [comment]
gh doesnt allow me to add you as a reviewer, so ill just tag here. @jgzuke

**Comment by pvinis** [comment]
tested locally with `npm run package-extension` and it works great.

**Comment by pvinis** [comment]
bundled up here. if you wanna use this PR's version of the extension:
- download the zip
- unzip to get a `.vsix`
- import to vscode using the command palette (cmd+shift+P) and the `Extensions: Install from VSIX` command.

[vscode-apollo-1.20.0.vsix.zip](https://github.com/apollographql/vscode-graphql/files/11501103/vscode-apollo-1.20.0.vsix.zip)

**Comment by donverduyn** [comment]
LGTM

File: syntaxes/graphql.js.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
           "name": "punctuation.definition.string.template.begin.js"
         }
       },
-      "end": "`",
+      "end": "`\\(?",
```
Comment (alessbell) [inline syntaxes/graphql.js.json]: Did you mean a closing parenthesis here?

Edit: pushed up this change.

```suggestion
      "end": "`\\)?",
```

**Review by alessbell — APPROVED** [review]
Thanks @pvinis!

**Pull request merged**